predictsignauxfaibles/__main__.py

```diff
@@ -1,80 +1,207 @@
-from datetime import datetime
+# pylint: disable=invalid-name
 import logging
 import sys
 
-from sklearn.metrics import balanced_accuracy_score
+from os import path
+import argparse
+import importlib.util
 
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from sklearn.metrics import balanced_accuracy_score
 from predictsignauxfaibles.pipelines import run_pipeline
+from predictsignauxfaibles.data import OversampledSFDataset, SFDataset
 
 sys.path.append("../")
-import models.default.model_conf as conf  # pylint: disable=wrong-import-position
 
 logging.basicConfig(format="%(asctime)s - %(levelname)s - %(message)s", level="INFO")
 
 # Mute logs from sklean_pandas
 logging.getLogger("sklearn_pandas").setLevel(logging.WARNING)
 
 
-def run():
+def load_conf(args):  # pylint: disable=redefined-outer-name
+    """
+    Loads a model configuration from a argparse.Namespace
+    containing a model name and a configuration filename
+    Args:
+        conf_args: a argparse.Namespace object containing attributes
+            model_name
+            model_conf_path
+    """
+    conf_filepath = path.join("models", args.model_name, args.model_conf_path + ".py")
```

Comment:
IMO a `model_name` should only ever have 1 conf file (always named `model_conf.py`)

Reply:
✅